### ISSUE TITLE
Track java code coverage

### ISSUE THREAD

**Issue by sherifnada** [airbytehq/airbyte#3684]
## Tell us about the problem you're trying to solve
As an Airbyte Java developer, I want to know what the code coverage of my unit tests is. This should be easily consumable/visible. I don't know exactly what that looks like, so suggestions are welcome.

**Comment by davinchia** [comment]
@jdpgrailsdev have we turned this on for OSS yet?

**Comment by jdpgrailsdev** [comment]
@davinchia Yes.  @alovew [added the required bits](https://github.com/airbytehq/airbyte/pull/12755) to not only generate the JaCoCo code coverage reports, but also echo out whether or not a module has met the minimum coverage requirements.  We started with the same levels from Cloud, so there is still some work to determine the correct levels that we want to enforce, the correct exclusions and whether or not we want to fail the builds if the levels are not achieved.

**Comment by davinchia** [comment]
Ok! Should we close this then?

**Comment by jdpgrailsdev** [comment]
@davinchia My two cents is yes.  This ticket seems to just mention having code coverage run and produce a visible report, which we now have.  We have [another ticket on the platform side to consolidate the reports into one central location](https://github.com/airbytehq/airbyte/issues/12180) to make it easier to find, but I think that work and the eventual work to enforce a coverage level are different/outside the scope of this ticket.

**Comment by davinchia** [comment]
Cool!
